requirements.txt

```diff
@@ -0,0 +1,7 @@
+python>=3.9.10
```

Comment:
Remove this? I get the following error on running `pip install -r requirements.txt`
```
ERROR: Could not find a version that satisfies the requirement python>=3.9.10 (from versions: none)
ERROR: No matching distribution found for python>=3.9.10
```